packages/mdc-list/README.md

```diff
@@ -1,49 +1,23 @@
-<!--docs:
-title: "Lists"
-layout: detail
-section: components
-excerpt: "Lists present multiple line items vertically as a single continuous element."
-iconId: list
-path: /catalog/lists/
--->
-
 # Lists
 
-<!--<div class="article__asset">
-  <a class="article__asset-link"
-     href="https://material-components-web.appspot.com/list.html">
-    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/lists.png" width="365" alt="Lists screenshot">
-  </a>
-</div>-->
-
-MDC List provides styles which implement [Material Design Lists](https://material.io/guidelines/components/lists.html) -
-"A single continuous column of tessellated subdivisions of equal width." Both single-line and two-line lists are
-supported (with three-line lists [planned](https://github.com/material-components/material-components-web/issues/31)).
-MDC Lists are designed to be accessible and RTL aware.
+MDC List provides styles which implement Material Design Lists - "A single continuous column of tessellated subdivisions of equal width." Both single-line and two-line lists are supported (with three-line lists planned). MDC Lists are designed to be accessible and RTL aware.
```

Comment:
Maybe for the first line:

MDC List provides styles which implement a single continuous column of tessellated subdivisions of equal width.